Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/datetimeinfo/ValidationTimeInfoFragment.kt

```diff
@@ -0,0 +1,30 @@
+package de.rki.coronawarnapp.covidcertificate.validation.ui.datetimeinfo
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.ValidationTimeInfoFragmentBinding
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.ui.popBackStack
+import de.rki.coronawarnapp.util.ui.viewBinding
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import javax.inject.Inject
+
+class ValidationTimeInfoFragment : Fragment(R.layout.validation_time_info_fragment), AutoInject {
+
+    @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
```

Comment:
this is not used and the viewmodel is not injected , do we need a dagger stuff at all ? I think the fragment only is enough , can we remove `ValidationTimeInfoFragmentModule ` and `ValidationTimeInfoViewModel `?

Reply:
We can sure leave them out, since they are currently doing nothing. I just created them as placeholders for the case that more logic is coming to the screen at some point.